instagram/src/App.js

```diff
@@ -21,14 +22,30 @@ class App extends Component {
     this.setState({ posts:dummyData }); 
```

Comment:
In order for the filtered posts to render, you will most likely need to set your state inside of a componentDidMount(), this is because it will trigger after render.  otherwise you will likely end up with a blank page at render time.